templates/support/Makefile

```diff
@@ -1,10 +1,34 @@
 # Support submodule makefile to be placed in: C:\Instrument\Apps\EPICS\support\{name}
 
-TOP = ../../extensions/master
```

Comment:
ah, really? I just copied the one from https://github.com/ISISComputingGroup/EPICS-CCD100/blob/master/Makefile which lives in the latter directory

Reply:
Yes we want this Makefile https://github.com/ISISComputingGroup/EPICS-CCD100/blob/master/Makefile to be copied into EPICS\support\{name}\master. But from this repo templates\support\Makefile is copied into EPICS\support\{name} rather than EPICS\support\{name}\master. The EPICS\support\{name}\master Makefile is created by running the perl support generator on line 54 of support_utils.py, I think we probably then want to append the ioctests to it. I will add this in now.

Reply:
ah i get you, cheers